lib/grand_central/model/subscriptable.rb

```diff
@@ -0,0 +1,16 @@
+require "grand_central/model"
+
+module GrandCentral
+  class Model
+    module Subscriptable
+      def self.included(base)
+        base.send(:alias_method, :merge, :update)
+      end
```

Comment:
Hmm, I think it had to do with some swizzling I was doing in my app, but can't remember now. It should be just defined like a regular method. I'll update that.